src/components/DateTimePicker/Internal/Utils/uiUtil.ts

```diff
@@ -23,48 +26,56 @@ export function waitElementReady(
     if (isVisible(element)) {
       callback();
     } else {
-      id = requestAnimationFrameWrapper(() => {
-        tryOrNextFrame();
-      });
+      if (canUseDom()) {
+        id = requestAnimationFrameWrapper(() => {
+          tryOrNextFrame();
+        });
+      }
     }
   }
 
-  tryOrNextFrame();
+  if (canUseDom()) {
+    tryOrNextFrame();
```

Comment:
if we expect to use this elsewhere, can we wrap it?  so the if is integrated

Reply:
Already got a check, most likely not to be used elsewhere. Will simplify.